src/graphic/text/rt_render.cc

```diff
@@ -308,18 +306,18 @@ class RenderNode {
 	TextClickTarget* click_target_;
 
 private:
-	Floating floating_;
+	Floating floating_{Floating::kNone};
 	UI::Align halign_;
 	UI::Align valign_;
-	int32_t x_, y_;
+	int32_t x_{0}, y_{0};
```

Comment:
I believe I have read somewhere that initialising and definition of multiple variables should be avoided.